src/test/java/bio/terra/service/job/JobServiceTest.java

```diff
@@ -324,32 +316,106 @@ private Matcher<JobModel>[] getJobMatchers(List<JobModel> jobModels) {
     return jobModels.stream().map(this::getJobMatcher).toArray(Matcher[]::new);
   }
 
-  // Submit a flight; wait for it to finish; return the flight id
+  /**
+   * Submit a flight with optional input parameters and wait for it to finish.
```

Comment:
I think this is a copy/paste error, I don't see where this submits a flight. Same with `expectedJobModel(int i)` below

Reply:
Yes, good shout!  I renamed these methods (ex. `runFlightAndReturnExpectedJobModel`).